modules/textures/src/lib/parsers/parse-dds.d.ts

```diff
@@ -0,0 +1,31 @@
+import { TextureLevel } from "@loaders.gl/textures/types/textrure";
+
+/**
+ * Check if data is in "DDS" format by its magic number
+ * @param data - binary data of compressed texture
+ * @returns true - data in "DDS" format, else - false
+ */
+export function isDDS(data: ArrayBuffer): boolean;
+
+/**
+ * Parse texture data as "DDS" format
+ * @param data - binary data of compressed texture
+ * @returns Array of the texture levels
+ */
+export function parseDDS(data: ArrayBuffer): TextureLevel[];
+
+/**
+ * DXT1 applicable function to calculate level size
+ * @param width - level width
+ * @param height - level height
+ * @returns level size in bytes
+ */
+export function getDxt1LevelSize(width: number, height: number): number;
```

Comment:
Why do the last two need to be exported? For testing?  Maybe add a separator stating that?

Reply:
These functions  are used in `parse-crunch.js`. Crunch files are "DXT" after decompression and we need for these functions to extract levels.